bigbluebutton-html5/imports/ui/components/actions-bar/container.jsx

```diff
@@ -40,5 +43,6 @@ export default withTracker(() => {
     hasBreakoutRoom: Service.hasBreakoutRoom(),
     meetingName: Service.meetingName(),
     users: Service.users(),
+    handleTakePresenter: () => takePresenterRole(Auth.userID),
```

Comment:
We can omit passing of `Auth.userID` here and only use it online 27 in the `takePresenterRole` function

`const takePresenterRole = () => { makeCall('assignPresenter', Auth.userID); };`